src/environment.js

```diff
@@ -13,5 +14,5 @@ export default function environment(name) {
   if (!envVar) {
     return null
   }
-  return envVar[0] === undefined ? envVar[1] : envVar[0]
+  return envVar[0] === undefined ? envVar[1] : envVar[0].trim()
```

Comment:
Why the `.trim()`?

Reply:
Just in case an environment or someone adds a space when passing these params, since none of them should have spaces around. Not really needed but I thought it might help someone to save an hour of their life :D

Reply:
Aaah I see! Yes, this can save a lot of frustration.  😂